### DISCUSSION TITLE
Project Roadmap

### DISCUSSION THREAD

**Discussion by aahnik** [aahnik/tgcf#43] [General]
```shell
_____              ________
__  /_______ _________  __/
_  __/_  __ `/  ___/_  /_  
/ /_ _  /_/ // /__ _  __/  
\__/ _\__, / \___/ /_/     
     /____/              
```

- [x] **Live Syncing**, i.e real-time forwarding of incoming messages 
- [x] **Granular control** over different aspects of forwarding.
- [x] **Basic Extensions**: Whitelist, Blacklist, Replacement, Filtering.
- [x] Advanced Custom use-case extensions: Watermarking and OCR.
- [x] **Documentation** and deployment guide.
- [x] **Video** Tutorial on YouTube.
- [x] **Bot UI** (buttons and commands) for controlling settings.
- [x] **Web App** for controlling settings.

Thanks to everyone who sponsored.

**Comment by anymeofu** [comment]
Wow Huge update ! Waiting for it

**Comment by RebbePod** [comment]
Sounds amazing.

Can the video / instructions include how to deploy on Heroku? (or is there a better way?)

**Reply by aahnik** [reply]
i will be releasing some of the features today

**Reply by RebbePod** [reply]
Oh wow that's great, I'm looking to create a simple forward from one channel (not owned by me), to another with a whitelist, so I'm looking forward!

**Reply by mamin123** [reply]
Awesome. Looking forward to have new feature released.

**Reply by aahnik** [reply]
today I released a new version #58 , documentation + videos coming soon.
@mamin123 @RebbePod @anymeofu

**Reply by anymeofu** [reply]
Wow So early 👍👀

**Reply by aahnik** [reply]
not all features released yet. but will be able to release everything before august 01.

**Reply by RebbePod** [reply]
Thank you, looks like I won't have to wait till August!

Does this new update allow forwarding from a channel that I'm just a subscriber to?

**Reply by anymeofu** [reply]
> Thank you, looks like I won't have to wait till August!
> 
> Does this new update allow forwarding from a channel that I'm just a subscriber to?

It already this before

**Comment by u-8d** [comment]
Congo for 100 stars 🤩

**Comment by jmbessa** [comment]
@aahnik have the new features already been released?

**Reply by tissole** [reply]
@jmbessa Yes, the features with checkboxes marked are available.

**Comment by myworkpool** [comment]
@aahnik  Hi,

For the bot UI issue,  will you focus on writing UI to create the config file?  or other methods?   I found the bot is really useful and looking to help.

**Comment by CA0721** [comment]
Does it support cloning groups with topics?
